src/libraries/System.Drawing.Primitives/ref/System.Drawing.Primitives.cs

```diff
@@ -332,40 +333,41 @@ public enum KnownColor
         Plum = 138,
         PowderBlue = 139,
         Purple = 140,
-        Red = 141,
-        RosyBrown = 142,
-        RoyalBlue = 143,
-        SaddleBrown = 144,
-        Salmon = 145,
-        SandyBrown = 146,
-        SeaGreen = 147,
-        SeaShell = 148,
-        Sienna = 149,
-        Silver = 150,
-        SkyBlue = 151,
-        SlateBlue = 152,
-        SlateGray = 153,
-        Snow = 154,
-        SpringGreen = 155,
-        SteelBlue = 156,
-        Tan = 157,
-        Teal = 158,
-        Thistle = 159,
-        Tomato = 160,
-        Turquoise = 161,
-        Violet = 162,
-        Wheat = 163,
-        White = 164,
-        WhiteSmoke = 165,
-        Yellow = 166,
-        YellowGreen = 167,
-        ButtonFace = 168,
-        ButtonHighlight = 169,
-        ButtonShadow = 170,
-        GradientActiveCaption = 171,
-        GradientInactiveCaption = 172,
-        MenuBar = 173,
-        MenuHighlight = 174,
+        RebeccaPurple = 141,
```

Comment:
This is a breaking change, enum members are just constants in IL and are returned as such:
https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEBLANgHwAEAmARgFgAoKwgZgAJDSlHj6Bhegbyvr8YZMWAQQwYo2YAFcMMACoBDKAHMYGAM70AsgAoAlN179jhAOz1R4yTPlLVGgHTtcC9eoDcRvgF8q3oA=

So something compiled against .NET 5 using `Red` will get `RebeccaPurple` in .NET 6

Reply:
See also https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEBLANgHwAEAmARgFgAoKwgZgAJDSlHj6Bhegbyvr8YZMW7CLmgBnegFkAFAEpuvfssIB2DqIkA6AEowAJgG4lfAL5Vz1SnXowAdgFcAthrFRxVHpWV799ALz0pGgm9ABCuA4wAfTEId78AOKw9jG0FkA, for an example.

Reply:
oh no, then what should i do about that? i will revert the changes. should i add rebeccapurple in the bottom with new number

Reply:
I would insert it at the bottom in the source assembly (as it needs to have the same value for `RebeccaPurple`) and then the regenerating the ref will ensure it has `= 175`

Reply:
Good catch, I completely missed the values were swapped, I thought it was only the order.

Reply:
i think this is now resolved with the latest commit i made.